apps/editor.planx.uk/src/pages/Preview/ApplicationViewer.tsx

```diff
@@ -0,0 +1,51 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import { FeeBreakdown } from "@planx/components/Pay/Public/FeeBreakdown/FeeBreakdown";
+import SummaryListsBySections from "@planx/components/shared/Preview/SummaryList";
+import { PrintButton } from "components/PrintButton";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React from "react";
+
+import StatusPage from "./StatusPage";
+
+const NoContentPage = () => (
+  <StatusPage
+    bannerHeading="Your form"
+    buttonText="Start a new form"
+  >
+    <Typography variant="body2">
+      No answers to display
+    </Typography>
+  </StatusPage>
+)
+
+const ApplicationViewer: React.FC = () => {
+  const [flow, passport, breadcrumbs, govUkPayment] = useStore(state => [state.flow, state.computePassport(), state.breadcrumbs, state.govUkPayment]);
+  const hasPaymentInfo = Boolean(govUkPayment);
+
+  // No answered questions - saving on first question
+  const noBreadcrumbs = Object.keys(breadcrumbs).length === 0;
+  if (noBreadcrumbs) return <NoContentPage />;
+
+  return (
+    <StatusPage bannerHeading="Your form" bannerText="Review your answers">
```

Comment:
Aware there is currently no clear record of your sessionId in the `SummaryList` below, should we add it to the header here?